doc/nrf/known_issues.rst

```diff
@@ -1837,7 +1838,7 @@ DRGN-11059: Front-end module API not implemented for SoftDevice Controller
 802.15.4 Radio driver
 =====================
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 KRKNWK-12482: Reception of correct frames can occasionally end in failure with error ``NRF_802154_RX_ERROR_RUNTIME``
```

Comment:
This is the limitation of 802.15.4 Radio Driver, not a known -issue. This shall be moved to the Limitations.

Reply:
Addressed in https://github.com/nrfconnect/sdk-nrfxlib/pull/809